backends/rapidpro/backend.go

```diff
@@ -156,6 +156,11 @@ func (b *backend) NewOutgoingMsg(channel courier.Channel, urn urns.URN, text str
 	return newMsg(MsgOutgoing, channel, urn, text)
 }
 
+// NewMsgOutgoing creates a new outgoing message from the given params
+func (b *backend) NewMsgOutgoing(channel courier.Channel, urn urns.URN, text string) courier.Msg {
```

Comment:
We already have the `NewOutgoingMsg` method with the same logic. So I guess this method can be removed.